crates/bevy_render/src/camera/camera.rs

```diff
@@ -17,7 +18,8 @@ pub struct Camera {
     pub depth_calculation: DepthCalculation,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone, Copy, Reflect, Serialize, Deserialize)]
+#[reflect_value(Serialize, Deserialize)]
 pub enum DepthCalculation {
```

Comment:
This is now a field in `OrthographicProjection`, so I had to add some extra derives, similar to the `ScalingMode` and `WindowOrigin` enums.

Reply:
Maybe a comment that `Distance` is generically applicable and `ZDifference` is an optimisation for an orthographic projection looking towards negative `-z` (i.e. its forward vector is `+z` because #1153, although don't include that in the comment)?